src/Platform.Handlers/src/Xamarin.Platform.Handlers/Handlers/View/AbstractViewHandler.cs

```diff
@@ -35,7 +35,7 @@ protected AbstractViewHandler(PropertyMapper mapper)
 
 		protected abstract TNativeView CreateNativeView();
 
-		protected TNativeView? TypedNativeView { get; private set; }
+		public TNativeView? TypedNativeView { get; private set; }
```

Comment:
Why is this public now? If you need it in the tests, you might just be able to use `View` and also test the type.

Reply:
I didn't really see any reason for it not to be public

If I have a slider handler that's using the generics
```C#
public partial class SliderHandler : AbstractViewHandler<ISlider, SeekBar>
```

It seems like I should have a property that's typed to the generic. It's like for our existing renderers how we have this property here

https://github.com/xamarin/Xamarin.Forms/blob/80a4dbfb1cd897f478c38d0aafa6a5d67940d2f3/Xamarin.Forms.Platform.iOS/ViewRenderer.cs#L50

Maybe View should only be exposed through an explicit interface and then the TypedView should be on the public API?

Reply:
I changed it back to protected for now

I'll propose a way to expose this property with a different PR